python/README.md

```diff
@@ -0,0 +1,149 @@
+# Welcome to the CUDA-Q Python API
```

Comment:
Did you mean to commit `python/README.md`? I assume you meant to just modify the .in file.